### DISCUSSION TITLE
Pure white MeshReflectorMaterial

### DISCUSSION THREAD

**Discussion by imjoshin** [pmndrs/drei#794] [Q&A]
Hello everyone!

I'm very new to three.js and I've been attempting to make a pure white mesh material with `MeshReflectorMaterial`. 
For my surface, I'm using:
```
<MeshReflectorMaterial
  blur={[250, 150]}
  color="#ffffff"
  depthScale={0.5}
  maxDepthThreshold={5}
  minDepthThreshold={0}
  mixBlur={1}
  mixStrength={8}
  metalness={0}
  resolution={1024}
  roughness={0.8}
/>
```

Here's the result so far:
![image](https://user-images.githubusercontent.com/5430474/157586967-7b1c0ac7-e861-4043-ad1d-b33c4dc9b1e4.png)

Thought increasing lighting or other parameters on the `MeshReflectorMaterial` itself wash out the reflections. Is it possible to get the result similar to `mirror={1}`, but with the black background being transparent or white?
![image](https://user-images.githubusercontent.com/5430474/157587145-f8bfd959-4ecf-419b-945e-5b84ff663d4a.png)

Thank you 😄

**Comment by mwmwmw** [comment]
I don't think transparent will work, but white is possible. 

mirror=1 will reflect the surrounding environment. So add a sphere that encompasses your scene, set it to side=BackSide and make it quite large. 

`MeshReflectorMaterial` will "see" that dome and reflect it. 

Use some fog to hide the boundary between the pure white reflection and the grey-ish dome.

`<fog attach="fog" color="white" near={10} far={20} />`

Here is an example
https://codesandbox.io/s/mirror-reflector-wvcwqz?file=/src/index.js

![image](https://user-images.githubusercontent.com/889851/163267936-6d6b4530-7415-4d4c-915d-65502c065fb1.png)
